stl/inc/xutility

```diff
@@ -1898,67 +1966,108 @@ public:
 
     static constexpr bool _Unwrap_when_unverified = _Do_unwrap_when_unverified_v<_BidIt>;
 
-    template <class _Src, enable_if_t<_Wrapped_seekable_v<_BidIt, _Src>, int> = 0>
+    template <class _Src, enable_if_t<_Wrapped_seekable_v<_BidIt, const _Src&>, int> = 0>
     constexpr void _Seek_to(const reverse_iterator<_Src>& _It) {
-        current._Seek_to(_It.base());
+        current._Seek_to(_It.current);
+    }
+
+    constexpr const _BidIt& _Get_current() const noexcept {

```

Comment:
nodiscard? Also, why is it important not to use `base()`?

Reply:
> nodiscard? 

👍 

> why is it important not to use `base()`?

It's one part "avoiding unnecessary copies", one part "consistency with necessary changes in `move_iterator`," and one part "avoiding iterator copies is the new normal since C++20 adds move-only single-pass iterators."